x-pack/packages/observability/observability_utils/es/utils/esql_result_to_plan_object.test.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ESQLSearchResponse } from '@kbn/es-types';
+import { esqlResultToPlainObjects } from './esql_result_to_plain_objects';
```

Comment:
The name of this file is "esql_result_to_plan_object.test.ts". Should be "plain_object".